src/state/slices/opportunitiesSlice/selectors/lpSelectors.ts

```diff
@@ -237,7 +248,7 @@ export const selectAggregatedEarnUserLpOpportunities = createDeepEqualOutputSele
           bnOrZero(aggregatedLpAssetBalance)
             .times(opportunityMetadata?.underlyingAssetRatiosBaseUnit[i] ?? '0')
             .div(bn(10).pow(bnOrZero(assets[assetId]?.precision)))
-            .toFixed(6)
+            .toFixed()
```

Comment:
this may break osmo? they only have 6 precision

Reply:
Tested against prod and didn't notice regressions - the rationale is we `.toString()`ed it either way - so that just makes sure the intermediary value is more correct

Reply:
plz update PR description to test osmo lp add/withdraw as we've changed significant stuff around LP in this PR